lapce-data/src/data.rs

```diff
@@ -2244,11 +2246,34 @@ impl LapceMainSplitData {
         editor_view_id
     }
 
+    fn get_name_for_new_file(&self) -> String {
+        let mut i = 0;
+        loop {
+            i += 1;
+            let potential_name = format!("new {}", i);
```

Comment:
Would it be better to be ```Untitled-n```?

Reply:
I was just following what Notepad++ does, it makes sense to me given that I have just hit Ctrl-N to create a new document. But I have no strong opinion, I'll change it if you want

Reply:
It's just I see Untitled more often.

Reply:
Changed.